src/navigation/sectionalTabs/sectionalTab.jsx

```diff
@@ -0,0 +1,116 @@
+import React, {
+    Component,
+} from 'react';
+import {
+    isFunction,
+} from 'lodash';
+import PropTypes from 'prop-types';
+import Classnames from 'classnames';
+import Typography from '../../dataDisplay/typography';
+import { BEM_NAVIGATION_TAB_ROOT_CLASS } from '../../global/constants';
+
+const propTypes = {
+    children: PropTypes.oneOfType([PropTypes.array, PropTypes.object, PropTypes.string]),
+    classNames: PropTypes.string.isRequired,
+    classes: PropTypes.shape({
+        sectionalTabLabel: PropTypes.string,
+        sectionalTabLabelSelected: PropTypes.string,
+    }),
+    id: PropTypes.string.isRequired,
+    onChange: PropTypes.func,
+    onClick: PropTypes.func,
+    originalKey: PropTypes.oneOfType([PropTypes.number, PropTypes.string]).isRequired,
+    selected: PropTypes.bool.isRequired,
+};
+
+const defaultProps = {
+    classes: {},
+    children: undefined,
+    onChange: undefined,
+    onClick: undefined,
+};
+
+class Tab extends Component {
```

Comment:
Please change the component name here from `Tab` to `SectionalTab`.

Reply:
Done